pkg/configprofiles/profiles.go

```diff
@@ -90,10 +90,6 @@ var virtClusterInitTasks = []autoconfigpb.Task{
 		/* nonTxnSQL */ []string{
 			// Enable multi-region abstractions in secondary tenants.
 			"SET CLUSTER SETTING sql.virtual_cluster.feature_access.multiregion.enabled = true",
-			// Disable range coalescing (as long as the problems related
-			// to range coalescing have not been solved yet).
-			"SET CLUSTER SETTING spanconfig.range_coalescing.system.enabled = false",
-			"SET CLUSTER SETTING spanconfig.range_coalescing.application.enabled = false",
```

Comment:
We might consider backporting this change to 23.2.

Reply:
I would feel better if we did, but we need to identify/write up a bug it is fixing first to do so.